iothub/device/src/ClientFactory.cs

```diff
@@ -28,7 +31,7 @@ internal class ClientFactory
         /// <param name="authenticationMethod">The authentication method that is used</param>
         /// <param name="options">The options that allow configuration of the device client instance during initialization.</param>
         /// <returns>InternalClient</returns>
-        public static InternalClient Create(string hostname, IAuthenticationMethod authenticationMethod, ClientOptions options = default)
+        internal static InternalClient Create(string hostname, IAuthenticationMethod authenticationMethod, ClientOptions options = default)
```

Comment:
Class is internal, and so these are effectively internal. I guess I finally got tired of the dissonance and misleading visibility spec.